### PR TITLE
Brig init blog post

### PR DESCRIPTION
**Pull request by willie-yao**
Added blog post introducing `brig init`

### CODE REVIEW

File: content/2021-07-19-brig-init.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Brigade 2 development efforts have been heavily focused on improving Brigade's UX. A significant pain point from Brigade 1.x that we've been intent on improving was the interactive, but onerous project creation process. Since the project's maintainers are big fans of "infrastructure as code" and because imitation is the sincerest form of flattery, Brigade 2 drew inspiration from Kubernetes and utilizes project definition files that are reminiscent of Kubernetes manifests (but aren't). Projects can be created and updated by submitting such definitions to Brigade.
+
+While project definitions as source offer an attractive improvement to Brigade, they aren't without their own all-new frustration. _Just how does one author one of these project definitions?_ We don't know about you, but most of us on the Brigade team can't write a Kubernetes manifest from scratch! We generally copy an existing one that's close to what we need and then modify it. But, we felt we could do better for Brigade 2's project definitions.
```
Comment (vdice) [inline content/2021-07-19-brig-init.md:14]: optional nit/style: perhaps `However, we felt we could do better for Brigade 2's project definitions.`

File: content/2021-07-19-brig-init.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Note that valid project names must contain only lowercase alphanumeric characters and dashes.
+
+The following files are generated in the `.brigade/` sub-directory:
+
+* `project.yaml`: This is the project definition file. Because no git integration was specified, a `brigade.ts` (TypeScript) script is _embedded_ within the project definition. The project definition and embedded script both contain commented examples of subscribing to and handling events.
+
+* `secrets.yaml`: This is a file for defining project secrets. `brig init` automatically adds this file to your `.gitignore`, and you should take great care to never commit this file to a public repository.
+
+* `NOTES.txt`: This is file contains next steps for for creating a project from your `project.yaml` file and defining project secrets using your `secrets.yaml` file.
```
Comment (vdice) [inline content/2021-07-19-brig-init.md:38]:
```suggestion
* `NOTES.txt`: This file contains next steps for creating a project from your `project.yaml` file and defining project secrets using your `secrets.yaml` file.
```

**Review by vdice — APPROVED** [review]
Reads great!  One grammar fix, otherwise LGTM.

File: content/2021-07-19-brig-init.md (hunk 1 above)
Comment (krancour) [inline content/2021-07-19-brig-init.md:14]:
```suggestion
While project definitions as source offer an attractive improvement to Brigade, they aren't without their own all-new frustration. _Just how does one author one of these project definitions?_ We don't know about you, but most of us on the Brigade team can't write a Kubernetes manifest from scratch! We generally copy an existing one that's close to what we need and then modify it. However, we felt we could do better for Brigade 2's project definitions.
```

**Comment by krancour** [comment]
Still want @karenhchu to approve this as well.

File: content/2021-07-19-brig-init.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Brig Init"
+slug: "2021-07-19-brig-init"
+description: "A deep dive into one of our favorite new features"
+date: "2021-07-19"
+author: "Kent Rancourt & Willie Yao"
+---
+# Brig Init
```
Comment (krancour) [inline content/2021-07-19-brig-init.md:8]: This header has to be removed. Hugo will automatically create a header from the title in the front matter. So as it stands now, we have double headers.

**Comment by krancour** [comment]
Also, please add this to the front matter:

```
summary: Last week, we announced the first beta release of Brigade 2. Today, we're taking a deep dive into an exciting new feature we shipped in that release: the Brigade CLI's new init command.
```

This compensates for issues that occur when the auto-generated summary contains links. The workaround is manually providing a summary that _doesn't_ have links.

**Pull request merged**